.eslintignore

```diff
@@ -1,2 +1,5 @@
 web-admin/build
-web-local/build
\ No newline at end of file
+web-local/build
+**/svelte.config.js
+
+web-common/src/components/modal/*.js
```

Comment:
ignored vendored files

Reply:
This comment would be good in the file itself :)